vsintegration/src/FSharp.Editor/Hints/FSharpInlineHintsService.fs

```diff
@@ -7,34 +7,38 @@ open System.ComponentModel.Composition
 open Microsoft.CodeAnalysis.ExternalAccess.FSharp.InlineHints
 open Microsoft.VisualStudio.FSharp.Editor
 open Microsoft.VisualStudio.FSharp.Editor.Telemetry
+open CancellableTasks
+open System.Threading.Tasks
 
 // So the Roslyn interface is called IFSharpInlineHintsService
 // but our implementation is called just HintsService.
 // That's because we'll likely use this API for things other than inline hints,
 // e.g. signature hints above the line, pipeline hints on the side and so on.
 
 [<Export(typeof<IFSharpInlineHintsService>)>]
-type internal RoslynAdapter [<ImportingConstructor>] (settings: EditorOptions) =
+type internal FSharpInlineHintsService [<ImportingConstructor>] (settings: EditorOptions) =
```

Comment:
Rename was mostly because I was trying to find it by partial name "HintsService" and VS couldnt, plus matches the interface. Let's factor it out once/when we will use it for other features then.